akka-stream-tests/src/test/resources/reference.conf

```diff
@@ -8,4 +8,5 @@ akka {
   akka.actor.warn-about-java-serializer-usage = false
 
   stream.materializer.debug.fuzzing-mode = on
+  stream.secret-test-fuzzing-warning-disable = 42
```

Comment:
WDYM? It is already exposed to users: http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0.3/scala/stream-testkit.html#fuzzing-mode